internal/backend/dispatcher/eventsworkflow.go

```diff
@@ -159,6 +159,52 @@ func (d *dispatcher) getEventSessionData(ctx context.Context, event sdktypes.Eve
 	return sds, nil
 }
 
+func (d *dispatcher) getSchedulerEventSessionData(ctx context.Context, event sdktypes.Event, triggerID sdktypes.TriggerID) ([]sessionData, error) {
+	z := d.z.With(zap.String("event_id", event.ID().String()))
+
+	if !event.IsValid() {
+		z.Error("could not find event")
+		return nil, sdkerrors.ErrNotFound
+	}
+
+	var sds []sessionData
+
+	trigger, err := d.services.Triggers.Get(ctx, triggerID)
+	if err != nil {
+		return nil, fmt.Errorf("get trigger: %w", err)
+	}
+
+	if trigger.EventType() != sdktypes.SchedulerEventTriggerType { // sanity
+		z.Error("unsupported trigger type, expected [scheduler]: ", zap.String("event_type", trigger.EventType()))
+	}
+
+	envID := trigger.EnvID()
+	if !envID.IsValid() {
+		z.Debug("no valid environment")
+		return nil, nil
+	}
+
+	deployments, err := d.services.Deployments.List(ctx,
+		sdkservices.ListDeploymentsFilter{State: sdktypes.DeploymentStateActive, EnvID: envID})
+	if err != nil {
+		z.Panic("could not fetch active deployments", zap.Error(err))
+	}
+
+	// FIXME: do we need test deployments?
```

Comment:
`FIXME/TODO` without ticket number

Reply:
@itayd  - I left this FIXME to ask you. Is it ok not to support test deployments?